src/Util/OWSDispatch.h

```diff
@@ -15,7 +15,7 @@ NS_ASSUME_NONNULL_BEGIN
  * Signal protocol session state must be coordinated on a serial queue. This is sometimes used synchronously,
  * so never dispatching sync *from* this queue to avoid deadlock.
```

Comment:
I wonder if we should elaborate this comment to note that all encryption and decryption should take place on this queue?